apps/tx-builder/src/components/forms/validations/validateAmountField.ts

```diff
@@ -0,0 +1,11 @@
+import { ValidateResult } from 'react-hook-form';
+
+import { isInputValueValid } from '../../../utils';
```

Comment:
Not to be done in this PR, but is better if we use the same approach of absolute routes if possible, we should check it